tests/console/ncurses.pm

```diff
@@ -24,7 +24,11 @@ sub run {
     select_console 'root-console';
     zypper_call 'in dialog';
     script_run 'dialog --yesno "test for boo#1054448" 3 20', 0;
-    assert_screen 'ncurses-simple-dialog';
+    assert_screen([qw(ncurses-simple-dialog ncurses-simple-dialog-broken-poo63026)]);
```

Comment:
It's possible to just define needles which have both tags "ncurses-simple-dialog" and "ncurses-simple-dialog-broken-poo63026" so you could not even need to change this `assert_screen` call.

Also, I wonder, is this really just a "test issue" and not a "product issue"? IIUC this should be a bug report in bugzilla.opensuse.org and that one should be referenced instead

Reply:
I think this is more a test issue than a product bug. @DimStar77, you may have some clue as it happens from time to time on x86 as well.

Reply:
Created a bug report: https://bugzilla.opensuse.org/show_bug.cgi?id=1183234
And using it as a reference instead of the poo#ticket.